proto/api/v2/report_service.proto

```diff
@@ -57,6 +57,7 @@ message VulnerabilityReportFilters {
     google.protobuf.Timestamp since_start_date = 6;
   }
   bool include_nvd_cvss = 7;
+  bool includeEpssProbability= 8;
```

Comment:
Thank you! This backend source of truth corresponds to recent derived frontend change:
https://github.com/stackrox/stackrox/pull/13806/files#diff-61bd63cb2f613dfd9bf9331c2f965c2f3ca7445db540b8a451a20d904063b70dR23